server/views/pages/prisonerSearch.njk

```diff
@@ -79,22 +79,41 @@
 
       {% set tableRows = [] %}
       {% for result in searchResults.content %}
+        {% set canViewProfile = user.permissions.canViewProfile(result) %}
+
         {% set photo %}
-          <img src="{{ routeUrls.prisonerPhoto(result.prisonerNumber) }}" alt="Photo of {{ result | nameOfPerson }}" class="app-prisoner-photo" />
+          {% if canViewProfile %}
+            <img src="{{ routeUrls.prisonerPhoto(result.prisonerNumber) }}" alt="Photo of {{ result | nameOfPerson }}" class="app-prisoner-photo" />
+          {% else %}
+            <img src="/assets/images/prisoner.jpeg" alt="Photo of {{ result | nameOfPerson }} is not available" class="app-prisoner-photo" />
+          {% endif %}
         {% endset %}
 
+        {% if result | isBeingTransferred %}
+          {% set location = result | prisonerLocation %}
+          {% set establishment = "N/A" %}
+          {% elif result | isOutside %}
```

Comment:
I think the indentation of this `elif` is off (at least it looks odd in GH)

Reply:
<3 YES this would annoy me so much if i spotted it later!